src/Dependencies/Collections/Microsoft.CodeAnalysis.Collections.Package.csproj

```diff
@@ -2,16 +2,13 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetPrevious);netstandard2.0</TargetFrameworks>
```

Comment:
I'm not sure why this had explicit references for net6.0 and netcoreapp3.1 since it appears this is effectively a source package?

Reply:
Agree. I'm not sure what the point of the multi-target here was other than maybe to get nullable warnings. But that didn't explain why two .NET TFMs.